packages/datadog_flutter_plugin/integration_test_app/android/gradle.properties

```
@@ -7,3 +7,4 @@
 org.gradle.jvmargs=-Xmx1536M
 android.useAndroidX=true
 android.enableJetifier=true
+kotlin_version=1.8.22
```

```suggestion
kotlin_version=1.8.22

```

minor: Newline at the EOF is missing in different files of this PR. It can be found in which ones by doing something like `git diff --staged | grep -B 10 newline`